pkg/cmd/roachtest/kv.go

```diff
@@ -22,11 +22,8 @@ import (
 )
 
 func init() {
-	runKV := func(t *test, percent, nodes int) {
-		ctx := context.Background()
-		c := newCluster(ctx, t, nodes+1, "--local-ssd")
-		defer c.Destroy(ctx)
-
+	runKV := func(ctx context.Context, t *test, c *cluster, percent int) {
```

Comment:
This doesn't have to happen here, but we should really stop putting the meat of these tests into `init` functions. Let's just move them out.